source/guides/maintenance/developer-workstation.rst

```diff
@@ -17,67 +17,204 @@ other Linux\* distributions.
-Use Table 1, *Developer Profiles*, to identify the *minimum
-required bundles* to get started developing based on your role or project.
-While your role may not neatly fit in one of these categories, consider
-Table 1 as a starting point.
-
-.. list-table:: **Table 1. Developer Profiles**
-   :widths: 20, 20, 20, 20
-   :header-rows: 1
-
-   * - |CL| Bundle
-     - *Internet of Things (IoT)*
-     - *System Administrator*
-     - *Client/Cloud/Web Developer*
-
-   * - :command:`editors`
-     - ✓
-     - ✓
-     - ✓
-
-   * - :command:`network-basic`
-     - ✓
-     - ✓
-     - ✓
-
-   * - :command:`openssh-server`
-     - ✓
-     - ✓
-     - ✓
-
-   * - :command:`webserver-basic`
-     -
-     - ✓
-     - ✓
-
-   * - :command:`application-server`
-     -
-     - ✓
-     - ✓
-
-   * - :command:`database-basic`
-     -
-     - ✓
-     - ✓
-
-   * - :command:`desktop-autostart`
-     - ✓
-     - ✓
-     - ✓
-
-   * - :command:`dev-utils`
-     -
-     -
-     - ✓
+After installing the minimum set of bundles required to get started, you can 
+add more bundles relevant to your specific use case.
+
+To run any process required for |CL| development, you can add the large 
+bundle :ref:`*os-clr-on-clr* <enable-user-space>`. However, you may want to deploy a leaner OS with only bundles relevant to your project.
+
+Use the **Developer Profiles** tabs to start installing *suggested bundles* based on your role or project. Consider these profiles as a starting point.
+
+..tip:: Add description on click on bundle to learn.... 
+
+
+.. tabs::
+
+   .. tab:: AI/ML Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Build machine learning applications with a full suite of libraries.
+           - `machine-learning-basic <https://clearlinux.org/software/bundle/machine-learning-basic/>`_
+
+         * - Build machine learning applications with PyTorch, an optimized tensor library for deep learning.
+           - `machine-learning-pytorch <https://clearlinux.org/software/bundle/machine-learning-pytorch/>`_
+
+         * - Build machine learning applications using Tensorflow, a library for numerical computation using deep neural networks.
+           - `machine-learning-tensorflow <https://clearlinux.org/software/bundle/machine-learning-tensorflow/>`_
+
+         * - Web-based, interactive tools for machine learning.
+           - `machine-learning-web-ui <https://clearlinux.org/software/bundle/machine-learning-web-ui/>`_
+
+         * - Machine learning Docker container.
+           - `machine-learning <https://clearlinux.org/software/docker/machine-learning/>`_
+
+         * - Pre-built Python libraries for Data Science.
+           - `python-extras <https://clearlinux.org/software/bundle/python-extras>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+   .. tab:: Computer Vision Engineer
```

Comment:
@ahkok Could you please review and advise on the bundles I included? My goal is to list at least 3 bundles per profile. Suggest removals too as appropriate.

Reply:
@ahkok. Do you have time to give feedback on this?

Reply:
Not my domain of expertise, no idea.

Reply:
@bryteise or @phmccarty, Can you recommend a reviewer for Computer Vision Engineer?

Reply:
@unrahul, Will you please review the suggested bundles below for accuracy in the role of `Computer Vision Engineer`?